services/worker-manager/schemas/v1/worker-full.yml

```diff
@@ -46,13 +46,31 @@ properties:
     type: string
     format: date-time
 
+  lastModified:
```

Comment:
This is out of scope of this PR, just something I remember looking at these: I wonder if we could open up some of the currently hidden useful fields in worker types? Especially now that we don't have direct access to the DB but need to debug things. Like some of the potentially useful `providerData`?

Reply:
`providerData` is intended to be used for internal-only information and sometimes might contain secrets etc. Perhaps we want another field called `tags` or something like that where we can define a format for data we want the ui to show?

Reply:
It's just that in that provider data we have stuff that's useful for debugging..... but I guess ease of debugging can be achieved in other ways as well